src/Symfony/Component/Serializer/Normalizer/PropertyNormalizer.php

```diff
@@ -129,6 +129,10 @@ protected function getAttributeValue($object, $attribute, $format = null, array
             $reflectionProperty->setAccessible(true);
         }
 
+        if (\PHP_VERSION_ID >= 70400 && !$reflectionProperty->isInitialized($object)) {
```

Comment:
Can we feature-detect this with `method_exists()` rather than checking for the PHP version?

Reply:
isn't integer comparison faster and more obvious than `method_exists`?